metatests/integration_test.sh

```diff
@@ -1,13 +1,37 @@
 #!/usr/bin/env bash
 
-## Script to run integration tests (see metatests/readme.md)
+## Script to run a single integration test (see metatests/readme.md)
 ##	* Intended to be run in a docker container simulating
 ##		Gradescope's autograder, as defined in metatests/Dockerfile
 
 cd /autograder
 
+## Set parameters based on function arguments
+if [[ $# -eq 0 ]]; then # zero arguments
+	## use defaults for the first integration test
+	source_dir="source/sample_submission"
+	required_files_dir="source/"
+elif [[ $# -eq 1 ]]; then # one argument
+	## same source and required_files directories
+	source_dir="$1"
+	required_files_dir="$source_dir"
+else	# two or more arguments
+	## different (non-default) source and required_files directories
+	source_dir="$1"
+	required_files_dir="$2"
+fi
+
+echo ""
+echo "###########"
+echo "## Running integration test for sample submission located at: $source_dir ##"
+echo "###########"
+echo ""
+
 # Copy the sample submission into the correct directory
```

Comment:
Could use clarification -- "correct" refers to the place the run_autograder script expects them.  Same with the next comment

Reply:
Agreed.  Will change.